tests/test_inputmedia.py

```diff
@@ -443,6 +443,72 @@ async def test_send_media_group_photo(self, bot, chat_id, media_group):
             mes.caption_entities == [MessageEntity(MessageEntity.BOLD, 0, 5)] for mes in messages
         )
 
+    async def test_send_media_group_throws_error_with_group_caption_and_individual_captions(
```

Comment:
the entries in `media_group` all have both a caption and one of parse_mode/caption_entities. Can you also check that the error is raised if one passe media that *only* have a parse_mode/caption_entities?

Reply:
Yes, I think that was what I was asking about earlier on, whether my test was enough.